wger/manager/views/workout.py

```diff
@@ -53,6 +57,44 @@
 logger = logging.getLogger(__name__)
 
 
+@login_required
+def export_workouts(request):
+        '''
+        Exports workouts
+        '''
+        # get all workouts
+        workouts = Workout.objects.filter(user=request.user)
+        # generate json data
+        data = serializers.serialize("json", workouts)
+        # create json response
+        response = HttpResponse(data, content_type='text/json')
+        # send response to data.json file
+        response['Content-Disposition'] = 'attachment; filename=data.json'
+        return response
+
+
+def import_workouts(request):
+    file = request.FILES['import_file']
+    data = file.read()
+    f = open('data.json', 'r')
```

Comment:
seems like you're reading from a hard-coded file here. is the `data` used in [line 78](https://github.com/andela/wger-lycan-clan/pull/10/files#diff-2923ec76a8e1150a8800e979bdee2de0R78) used anywhere?

Reply:
Also check usage of `with` when dealing with files